cephci.yaml.template

```diff
@@ -8,6 +8,10 @@ polarion:
   username: some_user
   password: some_password
 
+cdn_credentials:
```

Comment:
can you add couple comments saying for which sort of suites its a hard requirement ?

eg:
Provide these credentials for suites where we need to access cdn content like GAed container image registry log in etc..

Reply:
@XDsubhash Thanks for the review sure I will address this.